content/en/localstack/configuration.md

```diff
@@ -168,6 +168,37 @@ Some of the services can be configured to switch to a particular provider:
 | `KMS_PROVIDER` |  `moto` (default) and `local-kms` |
 | `SQS_PROVIDER` |  `moto` (default) and `elasticmq` |
 
+## Profiles
+
+LocalStack support configuration profiles which is stored in the `~/.localstack` config directory. A configuration profile is a set of environment variables stored in an `.env` file in the LocalStack config directory. Here is an example of what configuration profiles might look like:
+
+```sh
+% tree ~/.localstack 
+/home/username/.localstack
+├── default.env
+├── dev.env
+└── pro.env
+```
+
+Let's say you wish to have a look at the `dev.env` profile. You can do so by pushing the following command:
```

Comment:
If someone created an `.env` file I think we can assume that they know how to display it. I think we could reformulate this into "here is an example of what a specific environment profile looks like"